components/system/components/CodeBlock.js

```diff
@@ -36,13 +37,16 @@ const STYLES_PRE = css`
 const STYLES_CODE = css`
   box-sizing: border-box;
   background-color: ${Constants.system.pitchBlack};
-  font-family: ${Constants.font.monoCode};
+  font-family: ${Constants.font.code};
   color: ${Constants.system.gray};
   width: 100%;
   padding-left: 16px;
 `;
 
 export class CodeBlock extends React.Component {
+  componentDidMount() {
+    Prism.highlightAll();
+  }
```

Comment:
Nit: Make sure we add some white space here before functions to keep the code clean.